Misc/NEWS.d/next/Library/2023-09-10-23-35-20.gh-issue-109245.yCVQbo.rst

```diff
@@ -0,0 +1,3 @@
+Simplified ``ensurepip`` to stop assuming that it can provision multiple
+wheels. The refreshed implementation now expects to only provision
+a ``pip`` wheel and no other distribution packages.
```

Comment:
I don't think that a Changelog entry is needed. For end users, there is no visible change.

Reply:
Don't the downstream packagers deserve it?

Reply:
@vstinner would you apply the corresponding labels, then?